site/visitor-console/src/components/Footer.tsx

```diff
@@ -0,0 +1,11 @@
+const Footer = () => {
+  return <div className="text-muted w-full mt-5 fs-6 text-center">
+        This site is an open-source AWS Capstone project.
```

Comment:
Can we tweak this language to keep the focus on the students? Amazon's sponsorship of this project is incidental - the real work on this site has been done by students, for students!

```suggestion
        This site is developed by students in Clemson's School of Computing Capstone.
```

Reply:
Done! I took your suggested text and redeployed to my dev site.

https://d1kquov5x5h6vg.cloudfront.net/